src/App.tsx

```diff
@@ -1,34 +1,103 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api/api';
+import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { Selected } from './types/Selected';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(true);
+  const [modal, setModal] = useState(false);
+  const [id, setId] = useState(0);
+  const [userId, setUserId] = useState(0);
+  const [error, setError] = useState('');
+  const [selectedOption, setSelectedOption] = useState('all');
```

Comment:
Use Enum instead of the hardcoded value. You can just combine it with the default case